src/gridtools/clang/timer.hpp

```diff
@@ -0,0 +1,106 @@
+/*
+  GridTools Libraries
```

Comment:
can we use GT-code like this, how does licensing work?

Reply:
and if we have it as internal, we should use GTClang-Headers. If not, does it make sense to put it into an external folder?